platforms/hyperledger-fabric/charts/orderernode/templates/deployment.yaml

```diff
@@ -52,6 +52,8 @@ spec:
         args:
         - |-
           #!/usr/bin/env sh
+          if [ {{ $.Values.consensus.name }} == kafka ]
```

Comment:
Shouldn't this have ''?

Reply:
I dont think we need quotes (as it is not the first character of the string.
I have tested both kafka and raft and both seems fine with this